docs/guides/vault/interact-with-contracts.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: 'Interact With Contracts'
+sidebar_position: 3
+---
+
+# Interact With Contracts
+
+Similar to our **[previous guide](#)** on setting data on the Vault, in this guide we will be learning how to **interact with other contracts** through the execute function on the Vault.
```

Comment:
same being picky here but would probably say "through the Vault's execute function". @CallumGrindle ?